geo/src/algorithm/intersects/mod.rs

```diff
@@ -100,7 +100,7 @@ where
 // Helper function to check point lies inside rect given by
 // bounds.  The first bound need not be min.
 #[inline]
-fn point_in_rect<T>(value: Coord<T>, bound_1: Coord<T>, bound_2: Coord<T>) -> bool
+pub fn point_in_rect<T>(value: Coord<T>, bound_1: Coord<T>, bound_2: Coord<T>) -> bool
```

Comment:
Can this be made `pub crate`?

Reply:
doesn't appear so on naive try

Reply:
See https://github.com/georust/geo/pull/975/files#diff-202b9782bca6975a5e99f3e0264722b95db4f2caf099f8b79f8be015ec947f16R103